content/custom_resources/partials.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Partials"
+gh_repo = "chef-web-docs"
+
+[menu]
+  [menu.infra]
+    title = "Partials"
+    identifier = "chef_infra/cookbook_reference/custom_resources/partials"
+    parent = "chef_infra/cookbook_reference/custom_resources"
+    weight = 20
++++
+
+[InfoQ article](https://www.infoq.com/news/2020/05/chef-infra-16/)
+
+Resoruce partials are a way of DRYing up resource code.
+
+If you have three resources all which require the proprties, `user` and `group` to be set. You can write the property definition in one file and include it in the three resources.
```

Comment:
Suggest: For example let's say we have three resources that all require the properties `user` and `group`. Instead of writing those as `property` definitions in each of the three custom resources, we can write them in one file: